src/Modules/UI/Dnn.PersonaBar.UI/admin/personaBar/scripts/main.js

```diff
@@ -428,16 +428,35 @@ require(['jquery', 'knockout', 'moment', '../util', '../sf', '../config', './../
                     }
                 }
             },
+            /**
+             * in case path is an array, it is expected to be a sorted list of dependent sources.
+             * 
+             * @param path string|array
+             */
             loadBundleScript: function (path) {
-                if (path.indexOf('cdv=') === -1) {
-                    path += (path.indexOf('?') > -1 ? '&' : '?') + 'cdv=' + config.buildNumber;
+
+                function ajax(url, build) {
+                    var isArray = typeof url === 'object' && url.constructor === Array;
+                    $.ajax({
+                        dataType: "script",
+                        cache: true,
+                        data: {
+                            cdv: build
+                        },

```

Comment:
I'm not convinced. Previously the `cdv` param was added to URL's existing query string. Are you sure that doing the call like this won't lose this information if already present on the `path` variable?

Reply:
$.ajax will take care of building a proper GET request including all attributes of "data".